### PR TITLE
{2023.06}[foss/2021b] QuantumESPRESSO V6.8

### PR DESCRIPTION
**Pull request by TopRichard**

### CODE REVIEW

**Comment by TopRichard** [comment]
bot: build repo:eessi-2023.06-software arch:x86_64/generic

**Comment by boegel** [comment]
We should get #283 deployed first, since there's trouble there with OpenBLAS...

**Comment by TopRichard** [comment]
bot: build repo:eessi-2023.06-software arch:x86_64/generic

**Comment by TopRichard** [comment]
bot: build repo:eessi-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi-2023.06-software arch:x86_64/intel/skylake_avx512
bot: build repo:eessi-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi-2023.06-software arch:aarch64/generic
bot: build repo:eessi-2023.06-software arch:aarch64/neoverse_v1
bot: build repo:eessi-2023.06-software arch:aarch64/neoverse_n1

File: eessi-2023.06-eb-4.7.2-2021b.yml [hunk 1]
```diff
@@ -10,3 +10,4 @@ easyconfigs:
   - OpenMPI-4.1.1-GCC-11.2.0
   - gompi-2021b
   - FlexiBLAS-3.0.4-GCC-11.2.0.eb
+  - QuantumESPRESSO-6.8-foss-2021b.eb
```
Comment (boegel) [inline eessi-2023.06-eb-4.7.2-2021b.yml:13]: @TopRichard It won't make a difference in terms of what gets built, but I would add a separate entry for `foss-2021b` itself, as a "checkpoint" in the easystack file:

```suggestion
  - foss-2021b.eb
  - QuantumESPRESSO-6.8-foss-2021b.eb

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
all tarballs staged for ingestion

**Review by TopRichard — COMMENTED** [review]
Agreed

**Review by boegel — APPROVED** [review]

**Pull request merged**